src/main/java/thePackmaster/powers/serpentinepack/TaintedPower.java

```diff
@@ -0,0 +1,48 @@
+package thePackmaster.powers.serpentinepack;
+
+import com.evacipated.cardcrawl.mod.stslib.powers.interfaces.InvisiblePower;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.animations.VFXAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.actions.common.GainBlockAction;
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.actions.watcher.ChangeStanceAction;
+import com.megacrit.cardcrawl.actions.watcher.NotStanceCheckAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.PowerStrings;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import com.megacrit.cardcrawl.vfx.combat.EmptyStanceEffect;
+import com.megacrit.cardcrawl.vfx.combat.SmokeBlurEffect;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.powers.AbstractPackmasterPower;
+import thePackmaster.stances.serpentinepack.VenemousStance;
+
+public class TaintedPower extends AbstractPackmasterPower implements InvisiblePower {
+
+    public static final String POWER_ID = SpireAnniversary5Mod.makeID("TaintedPower");
+    private static final PowerStrings powerStrings = CardCrawlGame.languagePack.getPowerStrings(POWER_ID);
+    public static final String NAME = powerStrings.NAME;
+    public static final String[] DESCRIPTIONS = powerStrings.DESCRIPTIONS;
+
+    public TaintedPower(AbstractCreature owner, int amount) {
+        super(POWER_ID, NAME, PowerType.BUFF, false, owner, amount);
+    }
+
+    @Override
+    public void onApplyPower(AbstractPower power, AbstractCreature target, AbstractCreature source) {
+        if (power.type == PowerType.DEBUFF && !power.ID.equals("Shackled") && source == this.owner && target.hasPower(TaintedEnemy.POWER_ID) && !target.hasPower("Artifact")) {
+            if (target.hasPower(power.ID)){
+                target.getPower(power.ID).stackPower(power.amount);
+                target.getPower(power.ID).stackPower(power.amount * 2 * (target.getPower(TaintedEnemy.POWER_ID).amount-1));
```

Comment:
This is kinda jank. Would be better if you increased the amount pre-application. OnReceievePower from StSLib would allow for this.